nanomq/include/webhook_inproc.h

```diff
@@ -4,9 +4,9 @@
 #include "nng/supplemental/nanolib/conf.h"
 #include "nng/nng.h"
 
-#define WEB_HOOK_INPROC_URL "ipc:///tmp/webhook.ipc"
+#define HOOK_IPC_URL "ipc:///tmp/nanomq_hook.ipc"
```

Comment:
how about change file name also

Reply:
I have change it from webhook.ipc to nanomq_hook.ipc